pkg/okteto/client.go

```diff
@@ -173,3 +200,33 @@ func InDevContainer() bool {
 
 	return false
 }
+
+func getOrCreateKubeConfig(kubeConfigPath string) (*clientcmdapi.Config, error) {
+	var cfg *clientcmdapi.Config
+	_, err := os.Stat(kubeConfigPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			cfg = clientcmdapi.NewConfig()
+		} else {
+			return nil, err
+		}
+	} else {
+		cfg, err = clientcmd.LoadFromFile(kubeConfigPath)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return cfg, nil
+}
+
+func getContextName(cred *Credential, namespace, clusterName string) string {
+	contextName := ""
+	if namespace == "" {
+		// don't include namespace for the personal namespace
+		contextName = clusterName
+		namespace = cred.Namespace
```

Comment:
what's the objective of this assignment? `namespace` doesn't seem to be used anywhere below this.

Reply:
I copied it from SetKubeConfig which updates namespace. I'll resolve this